logout/index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://deploy-preview-7--sad-haibt-42112c.netlify.app/styles/main.css"
+    />
+    <link rel="stylesheet" href="logout.css" />
+    <link
+      rel="preload"
+      href="https://kit.fontawesome.com/43943f2889.js"
+      as="script"
+    />
+    <title>Home | Sanguine Store</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE store</div>
+      <label for="search" class="text-size-sm">
+        <input
+          type="search"
+          class="search-box padding-md border-rad-md"
+          name="search"
+          id="search"
+          placeholder="What are you looking for?"
+        />
+        Search
+      </label>
+      <ul class="list-style-none list-inline">
+        <li>
+          <a class="options link btn btn-secondary" href="/logout/logout.html"
+            >Logout</a
+          >
+        </li>
+        <li>
+          <a class="options link" href="">wishlist</a>
+        </li>
+        <li>
+          <a class="options link" href="">cart</a>
+        </li>
+      </ul>
+    </nav>
+    <main class="documentation">
+      <aside class="sidebar padding-lg">
+        <div class="header-container">
+          <span class="h4 text-wt-bold">Filters</span>
+          <span class="text-size-sm"
+            ><a href="" class="link text-dec-none">Clear All</a></span
```

Comment:
There should be some space between Filter and clear on the UI

Reply:
Yes, will try to fix these. Thanks.